shell/vue.config.js

```diff
@@ -61,33 +59,17 @@ module.exports = function(dir, _appConfig) {
   // If we have a local folder named 'shell' then use that rather than the one in node_modules
   // This will be the case in the main dashboard repository.
   if (fs.existsSync(path.join(dir, 'shell'))) {
-    SHELL = './shell';
     SHELL_ABS = path.join(dir, 'shell');
     COMPONENTS_DIR = path.join(dir, 'pkg', 'rancher-components', 'src', 'components');
   }
 
-  const babelPlugins = [
-    // TODO: Browser support
-    // ['@babel/plugin-transform-modules-commonjs'],
-    ['@babel/plugin-proposal-private-property-in-object', { loose: true }]
-  ];
-
-  if (instrumentCode) {
-    babelPlugins.push([
-      'babel-plugin-istanbul', { extension: ['.js', '.vue'] }, 'add-vue'
-    ]);
-
-    console.warn('Instrumenting code for coverage'); // eslint-disable-line no-console
-  }
-
```

Comment:
This appears to be made redundant by the `babelPlugins` variable we define down on line 412.